frontend/packages/knative-plugin/src/components/overview/__tests__/EventSinkServicesOverviewList.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
```

Comment:
build is still failing , try `import * as _ from 'lodash-es';` cc @karthikjeeyar

Reply:
I tried loadash-es but `yarn lint` flags use of loadash-es as error.